app/views/cases/sar_internal_review/_new_form_common.html.slim

```diff
@@ -0,0 +1,65 @@
+section
+  header
+
+  br
+  br
+    = f.text_field :subject_type, class: 'visually-hidden', 
```

Comment:
Does this need to be here? As it will show without styles. And with the other ones below. Can we use hidden inputs instead?

Reply:
It is here as the form needs to submit this value on error else the params are blank and the error page renders Nil in the table above if this is not here. There may be other ways of doing this but it seemed like the most straight forward.

Is there a reason to use the hidden input over this method i.e. better for accessibility etc.?

Reply:
Oh OK. Yes hidden input stays hidden if styles are off and won't be read out by the screen reader. But tbh it seems we're hiding and showing input fields in many places anyway. But good to minimise if easy, if not no big deal as we're not sticking to the rule in other places. (e.g. conditional radio reveals)

Reply:
I have added `hidden: true` to the fields and this works but the labels still requite a 'visually-hidden' class else they show up. I think this is a little bit of an improvement still though, so I'll go with it 👍